tools/autograd/gen_python_functions.py

```diff
@@ -228,8 +228,6 @@ def signature_original(f: NativeFunction) -> str:
         opname = str(f.func.name.name.base)
         if f.func.is_out_fn():
             opname += '_out'
-        # TODO: remove HACK
-        # I think we want to differentiate inplace functions here.. but we currently don't for the arg parser
```

Comment:
As Ed pointed out, this is actually the right behavior for the python arg parser.